src/Nethermind/Nethermind.Core/Specs/IReleaseSpec.cs

```diff
@@ -352,6 +357,7 @@ public interface IReleaseSpec : IEip1559Spec, IReceiptSpec
         public bool WithdrawalsEnabled => IsEip4895Enabled;
         public bool SelfdestructOnlyOnSameTransaction => IsEip6780Enabled;
 
-        bool MCopyIncluded => IsEip5656Enabled;
+        public bool IsBeaconParentBlockRootAvailable => IsEip4788Enabled;
```

Comment:
Not sure if this naming is helpful, also in a lot of places you use the original `IsEip4788Enabled`, so it is mixed between those two a lot.